yarn-project/aztec/src/cli/aztec_start_options.ts

```diff
@@ -0,0 +1,254 @@
+    {
+      flag: '--sandbox.enableGas',
+      description: 'Enable gas on sandbox start',
+      defaultValue: 'false',
+      envVar: 'ENABLE_GAS',
+      parseVal: val => ['1', 'true'].includes(val),
+    },
+  ],
+  API: [
+    {
+      flag: '--api-prefix',
+      description: 'Prefix for API routes on any service that is started',
+      defaultValue: '',
+      envVar: 'API_PREFIX',
+    },
+  ],
+  ETHEREUM: [
+    {
+      flag: '--l1-rpc-url <value>',
+      description: 'URL of the Ethereum RPC node that services will connect to',
+      defaultValue: 'http://localhost:8545',
+      envVar: 'ETHEREUM_HOST',
+    },
+    {
+      flag: '--l1-chain-id <value>',
+      description: 'The L1 chain ID',
+      defaultValue: '1337',
+      envVar: 'L1_CHAIN_ID',
+    },
+    {
+      flag: '--l1-mnemonic <value>',
+      description: 'Mnemonic for L1 accounts. Will be used if no publisher private keys are provided',
+      defaultValue: undefined,
+      envVar: 'MNEMONIC',
+    },
+  ],
+  'L1 CONTRACT ADDRESSES': [
+    {
+      flag: '--rollup-address <value>',
+      description: 'The deployed L1 rollup contract address',
+      defaultValue: undefined,
+      envVar: 'ROLLUP_CONTRACT_ADDRESS',
+    },
+    {
+      flag: '--registry-address <value>',
+      description: 'The deployed L1 registry contract address',
+      defaultValue: undefined,
+      envVar: 'REGISTRY_CONTRACT_ADDRESS',
+    },
+    {
+      flag: '--inbox-address <value>',
+      description: 'The deployed L1 -> L2 inbox contract address',
+      defaultValue: undefined,
+      envVar: 'INBOX_CONTRACT_ADDRESS',
+    },
+    {
+      flag: '--outbox-address <value>',
+      description: 'The deployed L2 -> L1 outbox contract address',
+      defaultValue: undefined,
+      envVar: 'OUTBOX_CONTRACT_ADDRESS',
+    },
+    {
+      flag: '--availability-oracle-address <value>',
+      description: 'The deployed L1 availability oracle contract address',
+      defaultValue: undefined,
+      envVar: 'AVAILABILITY_ORACLE_CONTRACT_ADDRESS',
+    },
+    {
+      flag: '--gas-token-address <value>',
+      description: 'The deployed L1 gas token contract address',
+      defaultValue: undefined,
+      envVar: 'GAS_TOKEN_CONTRACT_ADDRESS',
+    },
+    {
+      flag: '--gas-portal-address <value>',
+      description: 'The deployed L1 gas portal contract address',
+      defaultValue: undefined,
+      envVar: 'GAS_PORTAL_CONTRACT_ADDRESS',
+    },
+  ],
+  // We can't easily auto-generate node options as they're parts of modules defined below
+  'AZTEC NODE': [
+    {
+      flag: '--node',
+      description: 'Starts Aztec Node with options',
+      defaultValue: undefined,
+      envVar: undefined,
+    },
+    {
+      flag: '--node.archiverUrl <value>',
+      description: 'URL for an archiver service',
+      defaultValue: undefined,
+      envVar: 'ARCHIVER_URL',
+    },
+    // We don't need one for node right? It's only used for the node's archiver (if used)
```

Comment:
What about where it puts the world state?

Reply:
ah yes, correct. Thought that was part of the sequencer config